<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Software - Zhang Lab @ CMU</title>
  <meta name="description" content="Dr. Martin Zhang's lab webpage">
  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="/software/">
  <link rel="shortcut icon" type ="image/x-icon" href="/favicon.ico">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">

  <link rel="preconnect" href="https://player.vimeo.com">
  <link rel="preconnect" href="https://i.vimeocdn.com">
  <link rel="preconnect" href="https://f.vimeocdn.com">



<!-- Google Analytics (original) -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');

</script>

<!-- Global site tag (gtag.js) - Google Analytics 4 -->
<script async src="https://www.googletagmanager.com/gtag/js?id="></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', '');
</script>

<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','');</script>
<!-- End Google Tag Manager -->



</head>


  <body>

    <!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id="
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->

<nav class="navbar sticky-top navbar-expand-md navbar-dark bg-dark">
    <a class="navbar-brand" href="/">
     <img src="/favicon.ico" width="30" height="30" style="margin-right:5px" class="d-inline-block align-top" alt="">
      Zhang Lab @ CMU
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarColor02">
        <ul class="navbar-nav mr-auto">
        <ul class="navbar-nav">
          <li class="nav-item">
              <a class="nav-link" href="/">Home</a>
          </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/about">About</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/publications">Publications</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/research">Research</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/software">Software</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/people">People</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="/openings">Openings</a>
           </li> 
          
        </ul>
  </div>
</nav>



    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12 col-xs-12">
  <style>
img{
  border-radius: 10px;
}
iframe {
  width: 175px;
  display: inline;
  vertical-align:middle;
  <!-- margin-bottom:5px; -->
  <!-- margin-left:5px; -->
  <!-- border: 1px solid red; -->
}
.col-md-3 {
  margin:0;
  padding:0;
  margin-top:10px;
  margin-bottom:10px;
  display:block;
  overflow:hidden;
  text-align:center;
  display: table-cell;
  height: auto;
  float: none;
  background:white;
  border-radius:20px;
  <!-- border: 1px solid black; -->
}
</style>

<h2 id="software">Software</h2>

<div class="jumbotron">
  <div class="row align-items-end">
    <div class="col-md-12 col-sm-12">
      <h4><b> scDRS <a href="https://github.com/martinjzhang/scDRS" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a> </b></h4>

      <p>scDRS is a method that links scRNA-seq with polygenic risk of disease at individual cell resolution, as described in our paper “Polygenic enrichment distinguishes disease associations of individual cells in single-cell RNA-seq data” (<a href="https://www.nature.com/articles/s41588-022-01167-z" target="_blank">Zhang<sup>*</sup>, Hou<sup>*</sup> et al. 2022</a>).</p>

    </div>
  </div>
</div>

<div class="jumbotron">
  <div class="row align-items-end">
    <div class="col-md-12 col-sm-12">
      <h4><b> BanditPAM <a href="https://github.com/motiwari/BanditPAM" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a> </b></h4>

      <p>Popular k-medoids clustering algorithms, such as Partitioning Around Medoids (PAM), are prohibitively expensive in computation for large datasets. 
BanditPAM is a randomized version of PAM that returns the same results with high probability but is substantially faster. 
The algorithm is described in our paper “BanditPAM: Almost Linear Time k-Medoids Clustering via Multi-Armed Bandits” (<a href="https://proceedings.neurips.cc/paper_files/paper/2020/hash/73b817090081cef1bca77232f4532c5d-Abstract.html" target="_blank">Tiwari et al. 2020</a>).</p>

    </div>
  </div>
</div>

<div class="jumbotron">
  <div class="row align-items-end">
    <div class="col-md-12 col-sm-12">
      <h4><b> sceb <a href="https://github.com/martinjzhang/single_cell_eb" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a> </b></h4>

      <p>Empirical Bayes estimators for single-cell RNA-seq analysis, as described in our paper “Determining sequencing depth in a single-cell RNA-seq experiment” (<a href="https://www.nature.com/articles/s41467-020-14482-y" target="_blank">Zhang<sup>*</sup>, Ntranos<sup>*</sup> et al. 2020</a>).</p>

    </div>
  </div>
</div>

<div class="jumbotron">
  <div class="row align-items-end">
    <div class="col-md-12 col-sm-12">
      <h4><b> adafdr <a href="https://github.com/martinjzhang/adafdr" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a> </b></h4>

      <p>AdaFDR is a fast and covariate-adaptive method that learns adaptive p-value thresholds from covariates to improve power while controlling FDR. 
The method is described in our paper “Fast and covariate-adaptive method amplifies detection power in large-scale multiple hypothesis testing” (<a href="https://www.nature.com/articles/s41467-019-11247-0" target="_blank">Zhang et al. 2019</a>).</p>

    </div>
  </div>
</div>

<div class="jumbotron">
  <div class="row align-items-end">
    <div class="col-md-12 col-sm-12">
      <h4><b> cPCA <a href="https://github.com/abidlabs/contrastive" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a> </b></h4>

      <p>cPCA is a dimensionality reduction algorithm that identifies low-dimensional structures that are enriched in a dataset relative to comparison data. 
Applications include dicovering subgroups in biological and medical data. 
The method is described in our paper “Exploring patterns enriched in a dataset with contrastive principal component analysis” (<a href="https://www.nature.com/articles/s41467-018-04608-8" target="_blank">Abid<sup>*</sup>, Zhang<sup>*</sup> et al. 2019</a>).</p>

    </div>
  </div>
</div>

<!---
<div class="jumbotron">
<div class="row align-items-end">
<div class="col-md-12 col-sm-12">
<h4><b>Example Software</b></h4>
<a href="https://example.com" target="_blank"><button class="btn btn-success btn-sm">WEBSITE</button></a>
<a href="https://github.com" target="_blank"><button class="btn btn-info btn-sm">GIT</button></a>
<a href="/papers/example_proceeding.pdf" target="_blank"><button class="btn btn-danger btn-sm">PAPER</button></a> 

<b>Authors:</b>
<i>Example authors</i>

Example software description.

</div>
</div>
</div>
-->

</div>

      </div>
    </div>

    <br/>
<section id="footer">
<div class="container-footer">
  <div class="panel-footer">
	  <div class="row">
		<div class="col-sm-4">
		    <h5>About</h5>	
            <p>Martin Jinye Zhang<br/> Assistant Professor<br/> Computational Biology, SCS<br/> Carnegie Mellon University
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Contact</h5>	
            <p><a href="mailto:martinzh@andrew.cmu.edu" target="_blank"><i class="fa fa-envelope fa-1x"></i> Contact Martin via email</a> <br/> <a href="https://github.com/sbryngelson/academic-website-template"><i class="fa fa-github fa-1x"></i> Use this site as a template!</a>
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Coordinates</h5>	
            <p>Gates Hillman Center<br/> Computational Biology, SCS<br/> Carnegie Mellon University<br/> 5000 Forbes Avenue<br/> Pittsburgh, PA 15213
</p>
		</div>
	  </div>

      <center><p>&copy 2023 Zhang Lab @ CMU </p></center>
	</div>
  </div>
</div>

<script src="/assets/javascript/bootstrap/jquery.min.js"></script>
<script src="/assets/javascript/bootstrap/bootstrap.bundle.min.js"></script>


  </body>

</html>
