<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Research - Zhang Lab @ CMU</title>
  <meta name="description" content="Dr. Martin Zhang's lab webpage">
  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/research/">
  <link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4000/favicon.ico">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">

  <link rel="preconnect" href="https://player.vimeo.com">
  <link rel="preconnect" href="https://i.vimeocdn.com">
  <link rel="preconnect" href="https://f.vimeocdn.com">




</head>


  <body>

    <!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id="
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->

<nav class="navbar sticky-top navbar-expand-md navbar-dark bg-dark">
    <a class="navbar-brand" href="http://localhost:4000/">
     <img src="http://localhost:4000/favicon.ico" width="30" height="30" style="margin-right:5px" class="d-inline-block align-top" alt="">
      Zhang Lab @ CMU
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor02" aria-controls="navbarColor02" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarColor02">
        <ul class="navbar-nav mr-auto">
        <ul class="navbar-nav">
          <li class="nav-item">
              <a class="nav-link" href="http://localhost:4000/">Home</a>
          </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/about">About</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/publications">Publications</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/research">Research</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/software">Software</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/people">People</a>
           </li> 
          
           <li class="nav-item">
            <a class="nav-link" href="http://localhost:4000/openings">Openings</a>
           </li> 
          
        </ul>
  </div>
</nav>



    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12 col-xs-12">
  <style>
img{
  border-radius: 10px;
}
.col-md-3 {
  margin-top:10px;
  margin-bottom:10px;
  padding:0px;
  display:block;
  overflow:hidden;
  text-align:center;
  display: table-cell;
  background: white;
  border-radius: 20px;
  height: auto;
}
iframe {
  margin:0;
  padding:0;
  width: 175px;
  display: inline;
  vertical-align: middle;
}
</style>

<h2 id="research">Research</h2>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Polygenic enrichment in disease-critical cellular contexts</h4>

    <p>Heritable human diseases and complex traits often manifest in a highly cell-type-specific manner across the body.
Pinpointing these causal cell subpopulations are critical for functional follow-up and drug development. 
Integrating GWAS and functional genomics (e.g., scRNA-seq) has been demonstrated to be an effective approach. 
However, most existing works cannot capture the heterogeneity of disease association within classical cell types or distinguish causal cellular contexts from correlated (tagging) contexts.  with similar molecular patterns. 
We develop methods to capture the fine-grained diseases-associated cell populations and to tease apart the causal cellular contexts. 
<a href="https://www.nature.com/articles/s41588-022-01167-z" target="_blank">Single-cell disease relevance score paper</a> (Zhang<sup>*</sup>, Hou<sup>*</sup> et al. 2022).</p>

    <p><strong>Data types:</strong> GWAS (summary statistics); scRNA-seq; eQTL; single-cell eQTL; scATAC-seq; SHARE-seq; spatial transcriptomics.</p>
  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Genetic architecture of human diseases and complex traits</h4>

    <p>Understanding the genetic architecture of diseases and complex traits, namely the distribution of causal SNP effects across the genome, is a fundamental task with profound implications. 
For example, it can shed light on human evolutionary history and inform downstream tasks such as association testing, fine-mapping, polygenic risk prediction, and disease gene identification. 
Most existing works focus on the marginal distribution of SNP causal effect sizes, and we expand this endeavor in several directions. 
First, we move beyond the marginal distribution and develop methods to examine the correlation of causal effect sizes of SNP pairs, which may arise due to interaction with natural selection. 
Second, we investigate the genetic architecture involving both traits and molecular phenotypes, such as mediated trait heritability by gene expression. 
We develop methods to assess the mediated heritability by gene expression or protein abundance and partition it across cell types.</p>

    <p><strong>Data types:</strong> GWAS (summary statistics or individual genotype, e.g., UK Biobank); functional annotations (e.g., <a href="https://www.nature.com/articles/ng.3954" target="_blank">baseline-LD</a>); eQTL (e.g., GTEx); single-cell eQTL; pQTL (e.g., UK Biobank).</p>
  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Causal inference in genetics and genomics</h4>

    <p>Causal inference is a powerful statistical technique that can uncover the true underlying causal relationship between variables. 
However, its applications in genetics and genomics are often hindered by domain-specific structures such as a large number of variables, relatively smaller sample sizes, small effect sizes of genetic variants, complex model assumptions across molecular phenotypes, and different data collection methods. 
Therefore, methods need to be developed that carefully model these aspects to produce powerful and robust estimates. 
First, we develop integrative methods combining GWAS and molecular QTL data to estimate the causal relationship between genes/proteins and diseases. 
Second, we develop methods to disentangle causal from tagging relationships in gene expression experiments, such as differential expression analysis in RNA-Seq or scRNA-seq.</p>

    <p><strong>Data types:</strong> GWAS (summary statistics); eQTL (e.g., GTEx); single-cell eQTL; pQTL (e.g., UK Biobank); scRNA-seq; perturb-seq; massively parallel reporter assays (MPRAs).</p>

  </div>
</div>

<div class="jumbotron">
  <div class="col-md-12 col-sm-12">
    <h4>Machine learning and data science</h4>

    <p>We develop general machine learning and data science algorithms motivated by applications in genetics. 
<a href="https://proceedings.neurips.cc/paper_files/paper/2022/hash/08857467641ad82f635023d530605b4c-Abstract-Conference.html" target="_blank">Fast random forest paper</a> (Tiwari et al. 2022). 
<a href="https://proceedings.mlr.press/v151/ginart22a.html" target="_blank">Deployment monitoring for machine learning systems paper</a> (Ginart et al. 2022). 
<a href="https://elifesciences.org/articles/62293" target="_blank">Single-cell aging score paper</a> (Zhang et al. 2021). 
<a href="https://proceedings.neurips.cc/paper_files/paper/2020/hash/73b817090081cef1bca77232f4532c5d-Abstract.html" target="_blank">Fast k-medoid clustering paper</a> (Tiwari et al. 2020). 
<a href="https://www.nature.com/articles/s41467-019-11247-0" target="_blank">Covariate-adaptive multiple hypothesis testing paper</a> (Zhang et al. 2019). 
<a href="https://proceedings.mlr.press/v97/zhang19t.html" target="_blank">Adaptive Monte Carlo multiple testing paper</a> (Zhang et al. 2019). 
<a href="https://www.nature.com/articles/s41467-018-04608-8" target="_blank">Contrastive principal component analysis paper</a> (Abid<sup>*</sup>, Zhang<sup>*</sup> et al. 2018).</p>

  </div>
</div>

</div>

      </div>
    </div>

    <br/>
<section id="footer">
<div class="container-footer">
  <div class="panel-footer">
	  <div class="row">
		<div class="col-sm-4">
		    <h5>About</h5>	
            <p>Martin Jinye Zhang<br/> Assistant Professor<br/> Computational Biology, SCS<br/> Carnegie Mellon University
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Contact</h5>	
            <p><a href="mailto:martinjzhang@gmail.com" target="_blank"><i class="fa fa-envelope fa-1x"></i> Contact Martin via email</a> <br/> <a href="https://github.com/sbryngelson/academic-website-template"><i class="fa fa-github fa-1x"></i> Use this site as a template!</a>
</p>
		</div>

		<div class="col-sm-4">
		    <h5>Coordinates</h5>	
            <p>Gates Hillman Center<br/> Computational Biology, SCS<br/> Carnegie Mellon University<br/> 5000 Forbes Avenue<br/> Pittsburgh, PA 15213
</p>
		</div>
	  </div>

      <center><p>&copy 2023 Zhang Lab @ CMU </p></center>
	</div>
  </div>
</div>

<script src="/assets/javascript/bootstrap/jquery.min.js"></script>
<script src="/assets/javascript/bootstrap/bootstrap.bundle.min.js"></script>


  </body>

</html>
